lib/game.rb

```diff
@@ -0,0 +1,89 @@
+class Game
+  attr_accessor :player_1, :player_2, :board
+  
+  def initialize(player_1 = Player::Human.new("X"), player_2 = Player::Human.new("O"), board = Board.new)
+    @player_1 = player_1
+    @player_2 = player_2
+    @board = board
+  end
+
+
+  # WIN_COMBINATIONS constant
+  WIN_COMBINATIONS = [
+    [0,1,2], # Top row
+    [3,4,5],  # Middle row
+    [6,7,8],  # Bottom row
+    [0,3,6], # Left column
+    [1,4,7],  # Middle column
+    [2,5,8],  # Right column
+    [0,4,8],  # Left top diagonal
+    [6,4,2]  # Right top diagonal 
+  ]
+
+    #returns current player based on Board #turn_count
+  def current_player
+    if @board.turn_count % 2 == 0 
+      player_1 # return "X"
+    else
+      player_2 # return "O"
+    end
+  end
+
+  def over?
+    won? || @board.full? || draw?
```

Comment:
One of these expressions is redundant. If the board is full, either someone `won?` or the game is a `draw?`. Furthermore, the `draw?` method already calls `@board.full?`. You could use either `won? || @board.full?` or `won? || draw?`, depending on whether you value performance or readability more, respectively.